src/django/api/aws_batch.py

```diff
@@ -1,9 +1,9 @@
 import boto3
 import json
```

Comment:
The changes in this commit are the most far-reaching in the PR - I'd appreciate a careful double-check that I'm doing the right thing here. I think this is mostly a 1-1 replacement of the use of naive UTC datetimes with identical datetimes that have their `tzinfo` set to UTC, but I always get wary when dealing w/ dates & timezones.